scripts/060-astoria.sh

```diff
@@ -34,3 +34,13 @@ chown -R robot:robot /var/srobo/cache/
 mv /tmp/packer/astoria/astoria/* /usr/lib/systemd/system/
 chmod 644 /usr/lib/systemd/system/ast*d.service /usr/lib/systemd/system/astoria-udiskie.service
 systemctl enable astdiskd.service astmetad.service astoria-udiskie.service astprocd.service astwifid.service
+
+# Make pyenv python entrypoint globally available
+home="/home/$(id -un 1000)"
+if [ -d $home/.pyenv ]; then
+    python_bin_path="$(echo $home/.pyenv/versions/*/bin)"
```

Comment:
Suggestion: Instead of this, perhaps we should prepend the directory to the end of `$PATH`. This way we don't need to specify each binary manually, but we don't interfere with the system installed Python.

Reply:
That would involve templating a new path into every systemd file we have ... I think this is simpler.

Reply:
We could modify the system path globally in `/etc/environment`, but I agree this might be simpler. Shame it's something we have to do multiple times